builder/comp-builder.nix

```diff
@@ -90,6 +90,8 @@
 # LLVM
 , useLLVM ? ghc.useLLVM or false
 , smallAddressSpace ? false
+
+, prebuilt-depends ? []
```

Comment:
We should add a comment here. As I'm a bit fuzzy on what exactly prebuilt-depends are, I'm not sure I can come up with a good comment yet.

Reply:
added a new `Note [prebuilt dependencies]`, does that help?